x/superfluid/keeper/msg_server_test.go

```diff
@@ -418,7 +430,18 @@ func (suite *KeeperTestSuite) TestUnlockAndMigrateSharesToFullRangeConcentratedP
 	suite.App.GAMMKeeper.SetMigrationInfo(suite.Ctx, migrationRecord)
 
 	// Superfluid delegate the balancer pool shares
-	_, _, locks := suite.setupSuperfluidDelegations(valAddrs, []superfluidDelegation{{0, 0, 0, 9000000000000000000}}, denoms)
+	_, _, locks := suite.setupSuperfluidDelegations(valAddrs, []superfluidDelegation{{0, 0, 0, 9000000000000000000}}, []string{poolDenom})
+
+	// Create full range concentrated position (needed to give the pool an initial spot price and liquidity value)
+	suite.CreateFullRangePosition(clPool, fullRangeCoins)
```

Comment:
Yes, the problem arises when adding the CL LP share as a superfluid asset. This is because we need a spot price in order to determine the osmo equivalent multiplier. This means that we need to either create an initial position in the upgrade handler if we want to support superfluid / sf migration at upgrade time.

Reply:
Added an addition error case here that demonstrates this https://github.com/osmosis-labs/osmosis/pull/4885/commits/839ea2c08c129388e1ced8d68e91ed1cfa7c33b9

Reply:
Could you document this, please?

Reply:
I created an issue, as I think this is a bug rather than something that should be worked around. https://github.com/osmosis-labs/osmosis/issues/4978